nomad/local/integration-tests.nomad

```diff
@@ -34,6 +34,36 @@ variable "aws_endpoint" {
   description = "The endpoint in which we can expect to find and interact with AWS."
 }
 
+variable "analyzer_executor_integration_tests_tag" {
+  type        = string
+  default     = "dev"
+  description = "The tagged version of the analyzer_executor_integration_tests we should deploy."
+}
+
+variable "analyzerlib_integration_tests_tag" {
+  type        = string
+  default     = "dev"
+  description = "The tagged version of the analyzerlib_integration_tests we should deploy."
+}
+
+variable "engagement_edge_integration_tests_tag" {
+  type        = string
+  default     = "dev"
+  description = "The tagged version of the engagement_edge_integration_tests we should deploy."
+}
+
+variable "graphql_endpoint_tests_tag" {
+  type        = string
+  default     = "dev"
+  description = "The tagged version of the graphql_endpoint_tests we should deploy."
+}
+
+variable "rust_integration_tests_tag" {
+  type        = string
+  default     = "dev"
+  description = "The tagged version of the rust-integration-tests we should deploy."
+}
+
```

Comment:
While I appreciate the effort, I don't think there's ever any case where we'll be running a non-`dev` of integration tests.

Like, I could imagine a scenario where we swap out one of our services for an older version to see if that fixed an issue - but IDK why anybody would want a different tagged version of an integration test than the one on your machine.

Reply:
True, removing now